pkg/utils/annotations.go

```diff
@@ -61,3 +71,16 @@ func IsSoftwareRebootEnabled() (bool, error) {
 	}
 	return softwareRebootEnabled, nil
 }
+
+func GetWatchdogTimeout(node *v1.Node) time.Duration {
+	if node.Annotations == nil {
+		return 0
+	}
+
+	timeout, err := strconv.Atoi(node.Annotations[WatchdogTimeoutSecondsAnnotation])
+	if err != nil {
+		return 0
+	}
+
+	return time.Duration(timeout) * time.Second
```

Comment:
IIUC a use case where we can't get the value of the label isn't the expected flow.
As such, I think we should address it as an error.
Returning a 0 for this might be a risk of calculating a too low Safe time value.

Reply:
Yes, it's not the expected flow. But returning an error and retrying doesn't make sense IMHO, because the chance that anyone is setting the annotation on the unhealthy node later on is near zero. So we would never go on with remediation.
I agree though that 0 isn't the best choice. Do you have a suggestion for a safe value?

Reply:
I went with 60s, that's the maximum default timeout according to https://docs.kernel.org/watchdog/watchdog-parameters.html

Reply:
>But returning an error and retrying doesn't make sense IMHO

I Just wanted the error addressed and not ignored, I see you already did that by logging it :+1: 

>that's the maximum default timeout

That's what I had in mind as well, wasn't sure what's the exact number is though.